.changeset/dull-clouds-speak.md

```diff
@@ -0,0 +1,3 @@
+---
+---
```

Comment:
Because we export this utility for consumers, the package must be patched.

```suggestion
"@fuel-ts/account": patch
---
```

Reply:
Good catch, I had mistakenly thought the revert would have been in the same release as the original commit, but it was released.